bindata/network/network-metrics/003-prometheus_rules.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: network-metrics-with-network-name
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
+  namespace: openshift-monitoring
+  labels:
+    app: network-metrics
+    prometheus: k8s
+    role: alert-rules
+spec:
+  groups:
+    - name: network_with_name
+      rules:
+        - record: network:container_network_receive_bytes_total
```

Comment:
As per the docs https://prometheus.io/docs/practices/rules/#naming-and-aggregation, the recording rules have the following pattern:
> level:metric:operations

Reply:
> Left a few comments, but curious was there an enhancement for this, I am wondering about the background on this and more details and how this ties in with the network-metrics-daemon. Thanks!
> 
> cc @brancz

There is no enhancement, mostly because of my ignorance of needing one. I shared a doc with the network / multus team, and asked for validation on the monitoring team ml. @brancz replied to it.
The idea is to have a "dumb" (as in always 0 value) `pod_network_name_info` metric to "join" with the real network metrics to have a new metric with the network name.

Reply:
> As per the docs https://prometheus.io/docs/practices/rules/#naming-and-aggregation, the recording rules have the following pattern:
> 
> > level:metric:operations

I _tried_ to follow the pattern, but as I explained there is no real "operations" here.
How would you suggest to change this?